css/index.css

```
@@ -1,3 +1,284 @@
+    width: 880px;
+    height: 440px;
+    position: relative;
+}
+.left {
+    display: flex;
+    padding: 50px;
+    width: 440px;
+    text-align: center;
+    margin-top: 70px;
+    position: absolute;
+    flex-direction: column;
+    align-items: center;
+}
+.contents button {
+    border: 1px solid black;
+    background-color: white;
+    padding: 0 30px ;
+    font-family: 'Titillium Web', sans-serif;
+    font-weight: 600;
+    font-size: 16px;
+    width: 170px;
+    height: 40px; 
+}
+
+.right {
+    display: flex;
+    margin-top: 30px;
+    width: 440px;
+    position: absolute;
+    padding: 30px 0;
+    right: 0;
+	animation: spin 10s linear infinite;
+	animation-play-state: paused;
+}
+.right:hover {
+  animation-play-state: running;
+}
+@keyframes spin {
+100% {transform: rotate(1turn); }
+}
+h1 {
+	font-size: 76px;
+	font-weight: 400;
+}
+h2 {
+	font-weight: 400;
+}
+.main-content {
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+	margin-top: 45px;
+	padding: 30px 0px 30px 0px;
+}
+.top-left {
+
+    width: 420px;
+    display: inline-block;
+
+	margin-right: 12px;
+}
+.top-right {
+
+    display: inline-block;
+
+	width: 420px;
+	margin-right: 12px;
+}
+.image {
+	padding: 15px 0px 15px 0px;
+} 
+
+.bottom-left {
+    display: inline-block;
+
+	width: 273px;
+    margin-left: 5px;
+}
+.bottom-middle {
+    display: inline-block;
+	width: 273px;
+    margin-left: 5px;
+}
+.bottom-right {
+    display: inline-block;
+	width: 273px;
+    margin-left: 5px;
+}
+.footer {
+    margin-top: 30px;
+    margin-bottom: 40px;}
+.copyright {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    padding-left: 200px;
+}
+
+@media (max-width: 800px){
+    body {
+        background-color: lightblue;
+        width: 800px;
+      }
+    header {
+        width: 800px;
+    }
+    .right {
+        display: none;
+    }
+    .navigation {
+        width: 800px;
+    }
+    .contents {
+        width: 800px;
+    }
+    .top-left {
+        width: 380px;
+    }
+    .top-right {
+        width: 380px;
+    }
+    .bottom-left {
+        width: 200px;
+    }
+    .bottom-middle {
+        width: 200px;
+    }
+    .bottom-right {
+        width: 200px;
+    }
+}
+
+    @media (max-width: 500px){
+        background-color: pink;
+    }
+    header {
+        float: center;
+        padding: 15px;
+    }
+    .copyright {
+        padding-left: 30px;
+    }
+    body {
+        background-color: lightblue;
+        width: 500px;
+      }
+    header {
+        width: 500px;
+    }
+    .right {
+        display: none;
+    }
+    .navigation {
+        width: 500px;
+    }
+    .contents {
+        width: 500px;
+    }
+    .top-left {
+        width: 500px;
+    }
+    .top-right {
+        width: 500px;
+    }
+    .bottom-left {
+        width: 500px;
+    }
+    .bottom-middle {
+        width: 500px;
+    }
+    .bottom-right {
+        width: 500px;
+    }
+
+}
```

```suggestion
    @media (max-width: 500px){
        background-color: pink;
    }
    header {
        float: center;
        padding: 15px;
    }
    .copyright {
        padding-left: 30px;
    }
    body {
        background-color: lightblue;
        width: 500px;
      }
    header {
        width: 500px;
    }
    .right {
        display: none;
    }
    .navigation {
        width: 500px;
    }
    .contents {
        width: 500px;
    }
    .top-left {
        width: 500px;
    }
    .top-right {
        width: 500px;
    }
    .bottom-left {
        width: 500px;
    }
    .bottom-middle {
        width: 500px;
    }
    .bottom-right {
        width: 500px;
    }
 }
```  you need to have max-width and smaller widths on the children